media.css

```diff
@@ -0,0 +1,544 @@
+    background-image: url(img/header-img-375.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    height: 191px;
+  }
+
+  .top-content__block {
+    /* width: 355px;
+    height: 169px; */
+    padding-left: 20px;
+  }
+
+  .top-content__h1 {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 9px;
+  }
+
+  .link {
+    font-size: 12px;
+    line-height: 11px;
+  }
+
+  .hidden-768,
+  .hidden-375 {
+    display: none;
+  }
+
+  .catalog {
+    margin-left: 20px;
+    margin-right: 19px;
+  }
+
+  .catalog__title {
+    font-size: 20px;
+    line-height: 19px;
+  }
+
+  .catalog__item {
+    width: 281px;
+    height: 248px;
+  }
+
+  .catalog__item:not(:last-child)::after {
+    content: "";
+    display: none;
+  }
+
+  .catalog-item__image {
+    width: 165px;
+    height: 163px;
+    margin-bottom: 22px;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    padding-left: 0;
+  }
+
+  .catalog-item__title {
+    font-size: 15px;
+    line-height: 14px;
+  }
+
+  .catalog-item__description {
+    grid-template-columns: 1.5fr 1fr;
+    padding-left: 15px;
+    padding-right: 0;
+  }
+
+  .catalog-item__price {
+    font-size: 20px;
+    line-height: 19px;
+  }
+
+  .catalog-item__grey,
+  .catalog-item__black,
+  .catalog-item__red,
+  .catalog-item__orange,
+  .catalog-item__brown,
+  .catalog-item__cherry,
+  .catalog-item__blue {
+    height: 15px;
+    width: 15px;
+  }
+}
+
+@media (max-width: 320px) {
+  .wrapper {
+    width: 100%;
+  }
+
+  .header {
+    grid-template-columns: 1fr 1fr;
+    margin-bottom: 26px;
+  }
+
+  .header__menu,
+  .header__search,
+  .header__user-block,
+  .header__cart,
+  .header__language-block {
+    display: none;
+  }
+
+  .header__logo-image {
+    display: none;
+  }
+
+  .header__logo-image_adaptive {
+    width: 104px;
+    height: 22px;
+    display: flex;
+    align-items: center;
+    margin-left: 20px;
+  }
+
+  .header__menu-bar {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .menu-bar {
+    display: block;
+    width: 41px;
+    height: 17px;
+    margin-right: 20px;
+  }
+
+  .top-content {
+    background-image: url(img/header-img-375.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    height: 191px;
+  }
+
+  .top-content__block {
+    /* width: 355px;
+    height: 169px; */
+    padding-left: 20px;
+  }
+
+  .top-content__h1 {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 9px;
+  }
+
+  .link {
+    font-size: 12px;
+    line-height: 11px;
+  }
+
+  .hidden-768,
+  .hidden-375 {
+    display: none;
+  }
+
+  .catalog {
+    margin-left: 20px;
+    margin-right: 19px;
+  }
+
+  .catalog__title {
+    font-size: 20px;
+    line-height: 19px;
+  }
+
+  .catalog__items {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+  }
+
+  .catalog__item {
+    width: 281px;
+    height: 248px;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)